euclid.json

```diff
@@ -27,16 +27,32 @@
         "file_name": "token-key-2.p12",
         "alias": "token-key-2",
         "password": "password"
-      } 
+      }
     ]
   },
   "docker": {
     "default_containers": [
       "global-l0",
-      "metagraph-l0", 
-      "metagraph-l1-currency", 
-      "metagraph-l1-data", 
+      "metagraph-l0",
+      "metagraph-l1-currency",
+      "metagraph-l1-data",
       "monitoring"
     ]
+  },
+  "deployment": {
+    "remote_hosts": {
```

Comment:
Do you think we should just depend on the hosts.ansible.yml file for this instead of trying to duplicate that functionality in this config file? I can imagine other cases where the project would want to run 4 source nodes, for example, or customize the install process in different ways.

Reply:
One other idea would be to have the Ansible file locations as a config item here instead of the hosts, with the default locations preset on new installs. That would make the Ansible files more discoverable and allow people to create their own custom ones without editing the default ones.